app/store/env/fakebackend.js

```diff
@@ -2143,8 +2135,7 @@ YUI.add('juju-env-fakebackend', function(Y) {
       if (filename === 'icon.svg') {
         // This is a request for a local charm icon URL. Just return the
         // fallback icon hosted by charmworld.
```

Comment:
I think maybe this comment is out of date; looks like this is directly hosted within our assets?